docs/user_guide.md

```diff
@@ -12,5 +12,5 @@ The utilities are:
 
 ## Documentation for Previous Versions of NCEPLIBS-prod_util
 
-* [NCEPLIBS-prod_util Version 1.2.2](ver-1.2.2/index.html)
+* [NCEPLIBS-prod_util Version 2.0.14](ver-2.0.14/index.html)
```

Comment:
I don't think adding the 2.0.14 release here is necessary.

Reply:
Should it be something like this:
[NCEPLIBS-prod_util Version 2.0.14](/index.html)?